addons/microdagr/ACE_Settings.hpp

```diff
@@ -7,4 +7,12 @@ class ACE_Settings {
         isClientSettable = 0;
         values[] = {CSTRING(MapFill_None), CSTRING(MapFill_OnlyRoads), CSTRING(MapFill_Full)};
     };
+        class GVAR(waypointPrecision) {
```

Comment:
New settings should use `CBA_settings_fnc_init` directly

Reply:
You mean like here?

https://github.com/acemod/ACE3/blob/master/addons/advanced_fatigue/ACE_Settings.hpp
https://github.com/acemod/ACE3/blob/master/addons/advanced_fatigue/initSettings.sqf

Reply:
Yes, but since this is a new setting you would not use `movedToSQF`